css/social-icons.css

```
@@ -0,0 +1,77 @@
+.social-icons {
+  display: flex;
+  justify-content: center;
+
+  margin: 16px 0 0;
+}
+
+.social-icons__list {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+}
+
+.social-icons__item {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 44px;
+  height: 44px;
+  border-radius: 50%;
+
+  transition: all 0.2s linear;
```

be more specific in your transition property

```suggestion
  transition: background-color 0.2s linear;
```